ios/RocketChatRN.xcodeproj/project.pbxproj

```diff
@@ -1684,20 +1693,32 @@
 				CLANG_CXX_LIBRARY = "libc++";
 				CLANG_ENABLE_MODULES = YES;
 				CLANG_ENABLE_OBJC_ARC = YES;
+				CLANG_WARN_BLOCK_CAPTURE_AUTORELEASING = YES;
```

Comment:
I think you all these `CLANG*` are from your Xcode setup and shouldn't be commited.

Reply:
removed in latest commit